addons/pos_sms/data/sms_data.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" ?>
+<odoo>
+    <data noupdate="1">
+        <record id="sms_template_data_point_of_sale" model="sms.template">
+            <field name="name">POS: Sent Order Confirmation via Text</field>
+            <field name="model_id" ref="point_of_sale.model_pos_order"/>
+            <field name="body">
+                {{ (object.company_id.name + ': Your ' + object.pos_reference + ' is processed succesfully with amount ' + object.currency_id.symbol + ' ' + str(object.amount_total) + '. Use ' + object.pos_reference + ' for further reference')}}
```

Comment:
@vlst-odoo , `str(object.amount_total)` for this I guess we will get only one value which is already calculated in the `amount_total` field.

and also currently we are not sending messages through kiosk mode so i guess `object.pos_reference` this will always return eg. `Order  00004-001-0001` .
Let me know i miss something.

Reply:
> str(object.amount_total) for this I guess we will get only one value which is already calculated in the amount_total field.

my point is that `str(object.amount_total)` will not be correctly formatted. You have to format it as a monetary value ( with the currency etc )

> and also currently we are not sending messages through kiosk mode so i guess object.pos_reference this will always return eg. Order  00004-001-0001 .

The thing is that you should not rely on the fact that the string "Order" will always be "Order" or that it will be correctly translated, etc. You should make the template generic and not rely on an implementation detail